frontends/api/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@mit-open/api",
+  "packageManager": "yarn@3.2.0",
+  "sideEffects": false,
```

Comment:
`sideEffects: false` lets webpack tree-shake imports. E.g., if a a file exports `a` and `b`, and we only import `a`, then the code for `b` is omitted.